README.md

```diff
@@ -2,3 +2,171 @@
 A sample application composed of several services, written in Go.
 
 This application is a downloadable, interactive example that demonstrates the use of Datadog's Go auto-instrumentation ([DataDog/orchestrion](https://github.com/DataDog/orchestrion))
+
```

Comment:
Looks complete. Thanks! You might want Kari or Brett to give a pass on the wording. We also might want to make this a page on the DD public docs site in addition to being in the README.

Reply:
Sounds good, Jon! I talked to Kyle about single-sourcing this eventually. I'll set it up so the README connects to a page on the public docs title.

Kyle mentioned that we should wait to do this until this GAs at the end of the quarter. That'll be my plan unless anything changes. :)